app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,75 @@
+class RecipesController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @recipes = Recipe.all
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    @recipe = Recipe.new(recipe_params)
+    @recipe.user_id = current_user.id
+    if @recipe.save
+      flash[:notice] = 'Recipe successfully created'
+      redirect_to @recipe
+    else
+      flash[:notice] = 'Something went wrong'
+      render 'new'
+    end
+  end
+
+  def destroy
```

Comment:
[ OPTIONAL ] Users should only be allowed to delete their own recipes, not other users' recipes.